### PR TITLE
Mobile contact form

### PR DESCRIPTION
**Pull request by Fayob**
This is a portfolio setup project which is the mobile aspect of the project, the following includes what I did in the project.

I added html file (the markup document), these entails all the structure for the websites
I added style file, which entails the styles for the whole project
I setup the linters to check if some specific features were added to the website for better user experience
I added .gitignore file to ignore some file and folder
I added README file which entails the description about the project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+          required
+        />
+        <input
+          type="email"
+          name="user_email"
+          placeholder="Email address"
+          required
+        />
+        <textarea name="comment" maxlength="500" cols="20" rows="5">
+Enter text here</textarea
```
Comment (ShahierNashaat) [inline index.html]: - [ ] Kindly notice that the `textarea` tag should have `required` and `placeholder` attributes.

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @Fayob, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 

- Correct use of GitFlow ✔️
- Descriptive commit messages ✔️
- Professional looking pull request ✔️
- Your readme looks professional ✔️
- Good overall design ✔️
- Your code is clean and well documented ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Fayob) [reply index.html]: Sorry, am I going to remove the text in between the "textarea" tag if I use placeholder?

Comment (ShahierNashaat) [reply index.html]: Yes

Comment (Fayob) [reply index.html]: Alright, Thanks

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @Fayob   , 

Great work on making the changes requested by a previous reviewer 👏
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Github Flow.
✔️ Professional README
✔️ Good commit messages

## Optional suggestions

- [Optional ] Kindly, try to add the MIT LICENSE In README file. 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**